src/mjolnir/pbfgraphparser.cc

```diff
@@ -1366,7 +1366,8 @@ struct graph_callback : public OSMPBF::Callback {
           from_way_id = member.member_id;
         } else if (member.role == "to" &&
                    member.member_type == OSMPBF::Relation::MemberType::Relation_MemberType_WAY) {
-          restriction.set_to(member.member_id);
+          if (!restriction.to())
```

Comment:
This was a fix for when old editors allow people to enter 2 or more to roles for the end of a restriction.  We just take the first one and toss the rest.